### DISCUSSION TITLE
Session length

### DISCUSSION THREAD

**Discussion by yabenz** [acheong08/EdgeGPT#116] [General]
Hello,
First thanks for the share.

I wanted to ask you, I want to implement EdgeGpt in an article writing process. So I wanna do something like the following:

```
import asyncio
from EdgeGPT import Chatbot, ConversationStyle

bot = Chatbot(cookiePath='./cookie.json')

answer = bot.ask(prompt="Write an outline an article entitled: 'History of the Turing Test' ", conversation_style=ConversationStyle.creative))

section1 = bot.ask(prompt="Now expand on the first heading", conversation_style=ConversationStyle.creative))


```
etc...

So my question is first is my code valid  and second will it stay on the same discussion each time I send a (bot.ask) and untill when how can I know if the bot chat session has terminated so I can start another one?

Thank you!

**Comment by hrizal** [comment]
maybe it would be better to use 
[chatgpt](https://github.com/acheong08/ChatGPT)
